app/html/test.html

```diff
@@ -0,0 +1,38 @@
+<!-- This file is only for the test of Navbar and Footer.
```

Comment:
What is the purpose of this file?

Reply:
It's just locally test file. So it should be deleted. I missed it to be deleted.